### DISCUSSION TITLE
Published Workflows

### DISCUSSION THREAD

**Discussion by zhaolai-chu** [elsa-workflows/elsa-core#5329] [Q&A]
New to Elsa workflows.
I published a workflow and now I couldn't find it anywhere in the Studio, but it keeps executing based on the time interval I set.
Where are the published workflows located?
Thanks.

**Comment by glime-ai** [comment]
Published workflows in Elsa should typically be visible within the Elsa Dashboard under the "Workflow Definitions" section. If you cannot find your published workflow there, it could be due to a few reasons:

1. There might be a UI issue or a bug that is preventing the workflow from being displayed.
2. The workflow might have been deleted, but due to a bug or misconfiguration, the scheduled execution might still be occurring.
3. There could be a caching issue where the latest state of the workflow definitions is not being reflected in the UI.

To troubleshoot this issue, you can try the following steps:

- Refresh the Elsa Dashboard to see if the workflow appears.
- Check the database tables directly to see if the workflow definition still exists. The relevant tables are `WorkflowDefinitions`, `WorkflowInstances`, and `WorkflowExecutionLogRecords`.
- Look for any errors in the application logs that might indicate why the workflow is not being displayed.
- If you are using Elsa 2.x, ensure that you are looking at the correct version of the workflow, as Elsa supports versioning of workflow definitions.

If the workflow is indeed executing but not visible in the dashboard, and you wish to stop it, you may need to stop the application and investigate further to resolve the issue.

Please let me know if you need further assistance or if there are any specific error messages or behaviors you are observing that could help in diagnosing the problem.

This comment was generated by [Glime](https://glimelab.ai/?utm_source=dRbvTb7IhCSYiI&utm_medium=github).
